package.json

```diff
@@ -42,7 +42,11 @@
     "pre-commit": "^1.2.2",
     "promise": "^8.0.1"
   },
-  "dependencies": {},
+  "dependencies": {
+    "lodash.isfunction": "^3.0.8",
+    "lodash.isnil": "^4.0.0",
+    "lodash.pick": "^4.4.0"
+  },
```

Comment:
As rnvc uses [lerna](https://github.com/lerna/lerna), you can install dependencies with command `lerna bootstrap`.
These dependencies are not needed on root project. Can you remove these?